### DISCUSSION TITLE
External drive

### DISCUSSION THREAD

**Discussion by initialcommit810** [legendary-gl/legendary#168] [Q&A] [answered]
I have been wanting to know if you can install on an external flash/hard drive.

**Comment by derrod** [comment, marked as answer]
Yes, see the documentation on how to change your installation location.

**Reply by initialcommit810** [reply]
I am sorry but I can't seem to find it anywhere on the Github page.

**Reply by nutterthanos** [reply]
easy just type install legendary.exe or whatever you called it install game you want to install --base-path where you want to install
